src/components/Checkbox/Checkbox.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import Checkbox from "./Checkbox";
+
+<Meta
+  title="Form Elements/Checkbox"
```

Comment:
Make sure to utilize `getCategory("Checkbox")` here.

Reply:
Fixed!